docs/sphinx/_build/html/astro.dataframe.html

```diff
@@ -0,0 +1,138 @@
+
+<!DOCTYPE html>
+
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>astro.dataframe package &#8212; Astro Projects 0.6.0 documentation</title>
+    <link rel="stylesheet" type="text/css" href="_static/pygments.css" />
+    <link rel="stylesheet" type="text/css" href="_static/alabaster.css" />
+    <script data-url_root="./" id="documentation_options" src="_static/documentation_options.js"></script>
+    <script src="_static/jquery.js"></script>
+    <script src="_static/underscore.js"></script>
+    <script src="_static/doctools.js"></script>
+    <link rel="index" title="Index" href="genindex.html" />
+    <link rel="search" title="Search" href="search.html" />
+    <link rel="next" title="astro.ml package" href="astro.ml.html" />
+    <link rel="prev" title="astro package" href="astro.html" />
+   
+  <link rel="stylesheet" href="_static/custom.css" type="text/css" />
+  
+  
+  <meta name="viewport" content="width=device-width, initial-scale=0.9, maximum-scale=0.9" />
+
+  </head><body>
+  
+
+    <div class="document">
+      <div class="documentwrapper">
+        <div class="bodywrapper">
+          
+
+          <div class="body" role="main">
+            
+  <div class="section" id="astro-dataframe-package">
+<h1>astro.dataframe package<a class="headerlink" href="#astro-dataframe-package" title="Permalink to this headline">¶</a></h1>
+<div class="section" id="module-astro.dataframe">
+<span id="module-contents"></span><h2>Module contents<a class="headerlink" href="#module-astro.dataframe" title="Permalink to this headline">¶</a></h2>
+<p>Copyright Astronomer, Inc.</p>
```

Comment:
What does Airflow do to avoid displaying the license header as if it was the module's docstring? It may be worth addressing this as part of this PR.

Reply:
@utkarsharma2 as part of another PR review, @kaxil mentioned that we don't need the license headers per file:
https://github.com/astro-projects/astro/pull/172/files#r821220344
This could help solve the issue!